.gitignore

```diff
@@ -68,3 +68,8 @@ test.py
 
 .env
 *.ini
+MUSICALBUMS
+MUSICARTISTS
+MOVIES
+SHOWS
+MUSICPLAYLISTS
```

Comment:
These are the files created by the `generate_custom_slots.py` script. And honestly I wanted to put them in the `.gitignore` file too. But decided it was not worth a PR by itself.

That would avoid the following issue:
```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	MOVIES
	MUSICARTISTS
	MUSICPLAYLISTS
	SHOWS
```
Honestly I think they have their place there.